test/test.go

```diff
@@ -506,12 +506,7 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
-	// Ensure it's the absolute, symlink-less path we're returning
-	abs, err := filepath.EvalSymlinks(joined)
```

Comment:
Yep no problem, am away in Flanders this weekend so will fix it up as soon as I'm home 👍